openshift/auth.app.yaml

```diff
@@ -169,6 +169,11 @@ objects:
               configMapKeyRef:
                 name: auth
                 key: notification.serviceurl
+          - name: AUTH_CHE_SERVICEURL
+            valueFrom:
+              secretKeyRef:
```

Comment:
Don’t have to be a secret. Use config map instead.

Also we need to add it to https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.config.yaml (just as example for documentation porpoises).

And please add a check for this URL in configuration validation. Similar to https://github.com/fabric8-services/fabric8-auth/blob/master/configuration/configuration.go#L334

The validation will prevent deployments when the ConfigMap is not set.

We also have to create the corresponding CM in preview and prod.

Reply:
> We also have to create the corresponding CM in preview and prod. 
that's already in progress

> And please add a check for this URL in configuration validation. Similar to https://github.com/fabric8-services/fabric8-auth/blob/master/configuration/configuration.go#L334
hum, so having default values just for testing purpose is not a valid solution, then?

Reply:
@alexeykazakov I added the `che.serviceurl` CM entry in https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.config.yaml in b0bfeac